modules/local/verifyingmapq/main.nf

```diff
@@ -0,0 +1,38 @@
+/*
+This source file is adapted from a nf-core pipeline for SNVPhyl developed by
+Jill Hagey as a work of the United States Government, which was distributed as
+a work within the public domain under the Apache Software License version 2.
+
+This source file has been adapted to work within our pipeline.
+
+Please refer to the README for more information.
+*/
+process VERIFYING_MAP_Q {
+    tag ""
+    label 'process_single'
+    container = "staphb/snvphyl-tools:1.8.2"
+
+    input:
+    path(sorted_bams)
+
+    output:
+    path("mappingQuality.txt"), emit: mapping_quality
+    path("versions.yml"),       emit: versions
+
+    script:
+    def bam_line = ""
+
+    for (int i = 0; i <sorted_bams.size(); i++) {
+        bam_line += "--bam bam${i+1}=${sorted_bams[i]} "
```

Comment:
Can you use the file name as the key for the `bam${i+1}=${sorted_bams[i]}` statement instead of `bam${i+1}`?

This ultimately makes its way into a **mappingQuality.txt** report and using `bam${i+1}` here would mean the report contains:

```
bam3 : 7.29%
bam1 : 7.29%
bam2 : 7.29%
```

It should instead list the same names (which should be the same as the bam file name).

Reply:
Updated in https://github.com/phac-nml/snvphylnfc/pull/3/commits/6969a37e96983b1f68702c4264f616cb15b84208